files/disable_inactive_freeipa_users.sh

```diff
@@ -69,59 +69,76 @@ else
       sed --quiet 's/^\s*User login:\s*//p'
   )
 
+  # Grab the domain. Note that # --quiet means no printing unless the
+  # p command is used.
+  domain=$(resolvectl domain | sed --quiet 's/^[^:]*:\s\(.*cool\.cyber\.dhs\.gov\).*$/\1/p')
+
+  # Construct the LDAP searchbase
+  searchbase="cn=users,cn=accounts,dc="${domain//./,dc=}
+
   for user in $users; do
-    timestamps=$(ipa user-status "$user" \
-      |
-      # --quiet means no printing unless the p command is used
-      sed --quiet 's/\s*Last successful authentication:\s*//p')
-    # In the event that all timestamps are invalid (N/A) we don't want
-    # to disable users since we don't know the last time they
-    # authenticated.
+    # Do an LDAP query to get the timestamps corresponding to the
+    # user's creation time and the user's last authentication time.
+    ldapsearch_output=$(ldapsearch -Y GSSAPI -b "$searchbase" "uid=$user" createTimestamp krbLastSuccessfulAuth)
```

Comment:
If long options are not available would you instead mention briefly what the two flags do?

Reply:
Please see commit 9ed7d9a.

Reply:
Appreciated!